webrender/src/renderer.rs

```diff
@@ -4427,6 +4433,7 @@ impl<B: hal::Backend> Renderer<B> {
         frame_id: GpuFrameId,
         stats: &mut RendererStats,
         transit_to_present: bool,
+        _main_target: bool,
```

Comment:
we should have parameters gated by the feature, which became available recently in Rust
i.e.
```rust
#[cfg(not(feature "gl"))]
is_main_target: bool,
```

Reply:
I'm afraid adding that gate here would break our CI, because it's uses an older version of rust, and this feature is only available from `1.39 stable`.

Reply:
We should handle it specifically then, e.g.
```
#cfg(feature = "gl")]
let _ = main_target;
```

Reply:
Good idea, added this in https://github.com/szeged/webrender/pull/337/commits/140f94f9c1d0d24a416c7b95810c50dee8622cc2